docs/src/main/asciidoc/security-jwt.adoc

```diff
@@ -853,6 +853,28 @@ Please see the xref:security-openid-connect-client-reference.adoc#token-propagat
 [[integration-testing]]
 === Testing
 
+[[auto-generated-keys]]
+==== Automatic key generation
+
+When you have neither verification nor signing keys configured, the {extension-name} extension automatically provides an asymmetric RSA 2024 bit key pair for use in development and test modes.
```

Comment:
```suggestion
This extension generates an asymmetric RSA 2024 bit signing key pair in the dev and test modes if the verification key has not been configured.
Once the key pair is generated, the public RSA key is used to configure the `mp.jwt.verify.publickey` property, and the RSA private key is available to tests to sign tokens using `smallrye-jwt-build`, for example:
```

and give as a source java example:
```
import io.smallrye.jwt.build.Jwt;
String jwt = Jwt.upn("Alice").sign();
```

And then, probably, `See xref:smallrye-jwt-build.adoc#sign-claims for mode details.`